docs/docs.go

```diff
@@ -0,0 +1,297 @@
+// Code generated by swaggo/swag. DO NOT EDIT
```

Comment:
Everything inside `docs` seems auto-generated. How about we just add a `.gitkeep` so that the folder itself still exists, but update `gitignore` so that the generated files are not committed to the repository?

Reply:
They're auto-generated using the `swag` client. Those have all the documentation of all files. I think we could keep those versioned, since if someone downloads the repo and wants to figure out all the routes availabe they wouldn't need to run the command too. 

I also think it's good to keep those versioned because they will update others peoples swagger when eventually we created more actions and document then (withou the need of running the `swag` command too)

Reply:
Agree to disagree on this one. I think versioning auto-generated files is not ideal because they're prone to getting out of sync with the actual code, and cause noise when looking at diffs.

I'd prefer if we `gitignore` these and just add a note on the README on how to generate, or hook up the server's boot process to automatically generate the Swagger docs in case it's a quick process.

Reply:
It's not slow, but not superfast either. I can try something on bootstrapping.

Reply:
I think we're better of just ignoring the files then, because server boot-up needs to be quick so `buffalo dev` is decently usable.

Reply:
Just pushed the changes